internal/commands/chat.go

```diff
@@ -1,21 +1,45 @@
 package commands
 
 import (
+	"strconv"
+
 	"github.com/checkmarx/ast-cli/internal/wrappers"
 	"github.com/spf13/cobra"
 )
 
-const ConversationIDErrorFormat = "Invalid conversation ID %s"
+const (
+	ConversationIDErrorFormat  = "Invalid conversation ID %s"
+	AiGuidedRemediationEnabled = "scan.config.plugins.aiGuidedRemediation"
+)
 
-func NewChatCommand(chatWrapper wrappers.ChatWrapper) *cobra.Command {
+func NewChatCommand(chatWrapper wrappers.ChatWrapper, tenantWrapper wrappers.TenantConfigurationWrapper) *cobra.Command {
 	chatCmd := &cobra.Command{
 		Use:   "chat",
 		Short: "Chat with OpenAI models",
 		Long:  "Chat with OpenAI models regarding KICS or SAST results",
 	}
-	chatKicsCmd := ChatKicsSubCommand(chatWrapper)
-	chatSastCmd := ChatSastSubCommand(chatWrapper)
-
-	chatCmd.AddCommand(chatKicsCmd, chatSastCmd)
+	chatCmd.AddCommand(ChatKicsSubCommand(chatWrapper))
+	if aiGuidedRemediationEnabled(tenantWrapper) {
```

Comment:
This check won't be enough, we are already hidding the command in this [task](https://checkmarx.atlassian.net/browse/AST-36336) . Here I think we need to validate if the user has the ability to use the command and show an error message if not. So check aiGuidedRemediationEnabled before running the kics and sast command. But once more something to confirm with product.

Reply:
if we validate the aiGuidedRemediationEnabled after, the user will get this error 
"required flag(s) "chat-apikey", "sast-result-id", "scan-results-file", "source-dir" not set"
before the check of the setting
is it OK?